backend/de.metas.business/src/main/java/de/metas/inout/impl/InOutDAO.java

```diff
@@ -528,4 +529,40 @@ public ImmutableList<InOutId> retrieveShipmentsWithoutShipperTransportation(@Non
 				.listIds(InOutId::ofRepoId)
 				.asList();
 	}
+
+	@Override
+	public Stream<I_M_InOut> retrieveByQuery(@NonNull final InOutQuery query)
```

Comment:
u can keep it as it is now,
but usually we are calling those methods `streamByQuery`.

Reply:
I copied this method from master instead of creating a new one.